rotkehlchen/history/events/structures/types.py

```diff
@@ -25,11 +24,11 @@
     str | None,  # notes
     str,            # type
     str,            # subtype
+    str | None,  # extra_data
```

Comment:
SUPER DUPER nitpick: Rest seem to be aligned? Can you align this with the one above and one below.

Sorry I know ... but my OCD triggered :rofl: 

```suggestion
    str | None,      # extra_data
```